web/templates/setting/form.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= form_for @conn, @action, [as: :settings, method: :put], fn f -> %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for svc <- @system_variable_changesets do %>
+        <tr>
+          <td> <%= svc.data.key %> </td>
+
+          <%= inputs_for f, integer_to_atom(svc.data.id), fn ff -> %>
```

Comment:
If there's no immediate solution, we can leave it at that and later implement our own `inputs_for` to remove the atoms limitation. This PR is big enough that you can sort this detail out of it.